src/Repository/EventRepository.php

```diff
@@ -25,6 +26,11 @@ class EventRepository extends EntityRepository
         findOneByUuid as findOneByValidUuid;
     }
 
+    public function __construct(ManagerRegistry $registry, string $class = Event::class)
```

Comment:
why second arg?

Reply:
Because of CitizenActionRepository extends EventRepository

Reply:
and?

Reply:
EventRepository is the parent of CitizenActionRepository, than if you parent::__construct but you can't specify the class you have a little problem no ?

Reply:
oh I see, maybe in this case validate `$className` value?

Reply:
So weird